documentation/practices.md

```diff
@@ -56,7 +56,10 @@ In the `App.js` file, integration handlers for Google Analytics are pre-configur
 The `Analytics.js` file contains the functions used for logging page views with a React library for invoking Google Analytics functions (ReactGA), and enhanced e-commerce capabilities.
 
 ### Checkout Flow
-The check-out flow is pre-configured to log purchases through Google Analytics' basic e-commerce capabilities. Upon successful completion of a transaction, the storefront sends another request to submit the transaction and purchase amounts along with cart information. It is submitted under **Conversions > Ecommerce**, where Google Analytics analyzes the information. When you view a product, add a product to the cart, or remove a product from the cart, the product and cart pages submit this information for further analysis. 
+Along with the configurations for submitting transaction information to Cortex, the check-out flow is pre-configured to log purchases through Google Analytics’ basic e-commerce capabilities. When a customer completes a transaction, storefront provides the transaction details for Google Analytics to analyze the logged transaction or purchase amounts and cart information in the *Conversions > Ecommerce* section. 
```

Comment:
`Along with the configurations for submitting transaction information to Cortex, the storefront's check-out flow will submit purchases through Google Analytics’ enhanced e-commerce capabilities where they will be logged. When a customer completes a transaction, the storefront provides the transaction details for Google Analytics to analyze the logged transaction and purchase amounts and cart information in the *Conversions > Ecommerce* section. `

Reply:
Along with the configurations for submitting transaction information to Cortex, the storefront's check-out flow submits purchase details through Google Analytics’ enhanced e-commerce capabilities where the transactions are logged. When a customer completes a transaction, the storefront provides the transaction details for Google Analytics to analyze the logged transaction and purchase amounts and cart information in the *Conversions > Ecommerce* section.